pkg/machines/helpers.es6

```diff
@@ -119,6 +119,26 @@ const transform = {
         'custom': _("custom"),
         'host-model': _("host"),
     },
+    'networkType': {
```

Comment:
I guess these terms for `network type` are usually not translated. Has anyone experience with that?

Reply:
Well, in ukrainian, it is translated, but not all of them, for example UDP is still UDP.